### PR TITLE
Feat filters

### PR DESCRIPTION
**Pull request by rahulyadav139**
Demo: https://palasio-lane-store.netlify.app/

### CODE REVIEW

File: src/components/listing-page/filters/CarModels.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* .car-item .car__brand-input {
+  display: none;
+} */
```
Comment (Maianki) [inline src/components/listing-page/filters/CarModels.css:3]: please remove commented code

File: src/components/listing-page/filters/CarModels.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  margin-left: 0.5rem;
+  cursor: pointer;
+  font-size: 0.8rem;
+}
+/* .car-item input:checked + label + ul {
+  display: block;
+} */
+/* .car-item label {
+  cursor: pointer;
+} */
```
Comment (Maianki) [inline src/components/listing-page/filters/CarModels.css:34]: here also

File: src/components/listing-page/filters/filters.js [hunk 3]
```diff
@@ -4,21 +4,30 @@ import FilterByPrice from './FilterByPrice';
...
-      <FilterByPrice />
-      <FilterByCarModels products={props.products} />
-      <FilterByRatings />
+      <FilterByPrice onGetPrice={props.onGetPrice} price={props.price} />
+      <FilterByCarModels
+        onGetCarModels={props.onGetCarModels}
+        products={props.products}
+        carModels={props.carModels}
+      />
+      <FilterByRatings onGetStar={props.onGetStar} star={props.star} />
```
Comment (Maianki) [inline src/components/listing-page/filters/filters.js:26]: You can destructure these values. So you don't have to write props.propertyName every time.

**Review by Maianki — APPROVED** [review]
Great work. You have followed a good naming convention. And you are using BEM so great work there. Added a few suggestions. You can destructure your props apart from that your code is very clean.

File: src/components/listing-page/filters/filters.js (hunk 3 above)
Comment (rahulyadav139) [reply src/components/listing-page/filters/filters.js:26]: Sure, I will do that.

File: src/components/categories/Categories.js [hunk 4]
```diff
@@ -20,15 +21,18 @@ const Categories = props => {
 
           <div className="categories-tab">
             {collection.categories.map(category => (
-              <Link to={`/products/${collection.param}/${category.param}`}>
+              <Link
+                key={uuid()}
+                to={`/products/${collection.param}/${category.param}`}
+              >
                 <div key={category.id} className="image-card shadow">
                   <img className="image" src={category.imageRef} alt="" />
```
Comment (mehulsatardekar) [inline src/components/categories/Categories.js:29]: Make sure to add alt text for every images.

File: src/components/categories/Categories.js [hunk 5]
```diff
@@ -20,15 +21,18 @@ const Categories = props => {
...
                   <p className="">{category.categoryName}</p>
```
Comment (mehulsatardekar) [inline src/components/categories/Categories.js:30]: Remove className if u r not adding any classes to the tag

**Review by mehulsatardekar — APPROVED** [review]

File: src/components/categories/Categories.js (hunk 5 above)
Comment (rahulyadav139) [reply src/components/categories/Categories.js:30]: my bad. I will remove that.

File: src/components/listing-page/listing/Listing.js [hunk 6]
```diff
@@ -1,34 +1,63 @@
...
-          <i class="bi bi-filter-right"></i>Filter
+        <span className="text-grey heading-6">( showing 20 products)</span>
+        <button className="btn-filter">
+          <i className="bi bi-filter-right"></i>Filter
         </button>
       </div>
 
-      <div class="sorting-container flex gap align-center">
-        <div class="heading-6">Sort By:</div>
-        {/* <input type="radio" name="sort" id="popularity" />
```
Comment (keyurshah339) [inline src/components/listing-page/listing/Listing.js:17]: remove comments

**Pull request merged**